README.md

```diff
@@ -1,35 +1,113 @@
 [![PyPI](https://img.shields.io/pypi/v/fairdatapoint-client)](https://pypi.org/project/fairdatapoint-client/)
 [![Build_Test](https://github.com/fair-data/fairdatapoint-client/actions/workflows/build_test.yml/badge.svg)](https://github.com/fair-data/fairdatapoint-client/actions/workflows/build_test.yml)
-[![Codacy Badge](https://app.codacy.com/project/badge/Grade/240d21007f7f41259231141aba79ef1f)](https://www.codacy.com/gh/fair-data/fairdatapoint-client/dashboard?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=fair-data/fairdatapoint-client&amp;utm_campaign=Badge_Grade)
 [![Coverage Status](https://coveralls.io/repos/github/fair-data/fairdatapoint-client/badge.svg?branch=master)](https://coveralls.io/github/fair-data/fairdatapoint-client?branch=master)
-
+[![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?project=fair-data_fairdatapoint-client&metric=alert_status)](https://sonarcloud.io/dashboard?id=fair-data_fairdatapoint-client)
 
 # fairdatapoint-client
 
-The `fairdatapoint-client` is a simple and elegant Python package to work with
-REST APIs of FAIR Data Point.
+### Contents
+
+-   [Overview](#overview)
+-   [Installation](#installation)
+-   [Quick Tutorial](#Tutorial)
+-   [Issues & Contributing](#Issues-and-Contributing)
+-   [License](./LICENSE)
+
+## Overview
+
+fairdatapoint-client is a simple and elegant Python package to work with
+REST APIs of [FAIR Data Point](https://github.com/fair-data/fairdatapoint).
+
+The supported APIs are listed below:
```

Comment:
Is this library an answer to the question "How do I connect to the REST API of my favourite FDP" or to the question "How do I access my favourite FDP from Python"? In other words, is it important that it's a REST API? Or would it be better to introduce it as a library with which you can read and write catalogs, datasets and distributions in an FDP from Python?

Reply:
OK, sounds more user-friendly.